packages/roosterjs-editor-core/lib/corePlugins/utils/InlineEntityHandlers/isDelimiter.ts

```diff
@@ -0,0 +1,23 @@
+import { DELIMITER_AFTER, DELIMITER_BEFORE, DelimiterType } from './constants';
+import { Entity } from 'roosterjs-editor-types';
+import { getEntityFromElement } from 'roosterjs-editor-dom';
+
+export function isDelimiter(
+    ent: Element | Entity | null | undefined
+): [DelimiterType, Entity] | null {
+    if (!ent) {
+        return null;
+    }
+    let entity: Entity | null = (<Entity>ent).wrapper
+        ? <Entity>ent
+        : getEntityFromElement(<HTMLElement>ent);
+    if (entity) {
+        if (entity.type.indexOf(DELIMITER_AFTER) > -1) {
```

Comment:
Although I suggest we don't make the delimiters entity, but if we still do this, I suggest we have two separate entity type names, so that we just need to compare their type name directly